runtime/appruntime/metrics/aws/cloudwatch.go

```diff
@@ -156,3 +158,19 @@ func (x *Exporter) getClient() *cloudwatch.Client {
 	}
 	return x.client
 }
+
+func getSysMetrics(now time.Time) []types.MetricDatum {
```

Comment:
System metrics need an instance ID / name field.

In GCP we can run up to two instances during a deploy - so you'd get two metrics being returned; 1 for the old instance, 1 for the new instance.

Without labels for the instances, your gauges are going to be bouncing between two values, depending on which instance reported last.


Also when some customers scale up beyond 1 instance on purpose, this isn't going to be great without being able to view _per instance_

Reply:
That's already handled in GCP as we add `node_id` to the set of labels every metric has. It's not handled properly for AWS, tho.

I'll try and look for an instance ID we can attach to all metrics in AWS.